earth_enterprise/BUILD.md

```diff
@@ -0,0 +1,57 @@
+If you want to build, install, or run Portable server, see the [Portable Server page](https://github.com/google/earthenterprise/wiki/Portable-Server).
+
+# Building Earth Enterprise Fusion and Server
+
+Building is currently supported on 64-bit versions of Ubuntu 14.04 LTS, Ubuntu 16.04 LTS, RHEL 6, RHEL 7, and CentOS 7.
+
+
+## GEE 5.2.1 Build Prerequisites (All platforms) 
+
+1. Setup required Tools and Dependencies
+    * git 1.7.1+
+    * git lfs 
+    * gcc 4.8.x
+    * scons 2.0.x+
+    * python 2.6.x+ and Python 2.7.x+.  Python 3.0+ is not supported. 
```

Comment:
This should be an 'or' between the Python versions. Also, you may want to capitalize the names of the different software projects in this list.